components/arm/trossen/trossen.go

```diff
@@ -33,6 +33,15 @@ import (
 	rdkutils "go.viam.com/rdk/utils"
 )
 
+// This is an implementation of the Arm API for Trossen arm models
+// vx300s and wx250s ONLY (these codenames can be found at https://docs.trossenrobotics.com/interbotix_xsarms_docs/)
+// Specifications for vx300s (ViperX-300 6DOF):
+// https://docs.trossenrobotics.com/interbotix_xsarms_docs/specifications/vx300s.html
+// Specifications for wx250s (WidowX-250 6DOF):
+// https://docs.trossenrobotics.com/interbotix_xsarms_docs/specifications/wx250s.html
```

Comment:
Since we talked a lot about shortcomings of this implementation, can you add a comment about this driver being experimental.

Reply:
adding